content/en/apps/guides/forms/rdtoolkit-integration.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Rapid Diagnostic Toolkit Integration"
+linkTitle: "RDToolkit Integration"
+weight:
+description: >
+  Integration for the Rapid Diagnostic Toolkit (RDToolkit)
+keywords: RDToolkit RDT
+
+---
+
+_Introduced in v3.12.0_
+
+CHT has an integration with [Rapid Diagnostic Toolkit (RDToolkit)](https://github.com/dimagi/rd-toolkit) which is an Android app developed by Dimagi that facilitates the use of Rapid Diagnostic Test (RDT) in global health care. 
+The integration is only available in the CHT Android app and consist of two Enketo widgets (rdtoolkit-provision and rdtoolkit-capture) that are used in XForms to provision RDT and capture results.
+
+## Create the XForms
+It is required to create one form for provisioning RDT and one form for capturing RDT results, see [the documentation](https://docs.communityhealthtoolkit.org/apps/tutorials/app-forms/) for more details about app forms.
+
+### Provisioning RDT form
+It is required to add the following labels in the “appearance” tag:
+* `rdtoolkit-provision`: To attach the Enketo widget that connects with the RDToolkit app.
+* `rdtoolkit-action-btn`: To attach the action button that provisions RDT, this button will open the RDToolkit app.
+
+This form has the following mandatory fields:
+* `patient_uuid` | Text
+    * Patient's unique ID in the system.
+    * Auto filled when using the contact lookup field.
+* `patient_id` | Text
+    * Patient's representative ID, for example, can be the ID in the health facility's records.
+    * Auto filled when using the contact lookup field.
+* `patient_name` | Text
+    * Patient's first name and / or last name
+    * Auto filled when using the contact lookup field.
+* `rdtoolkit_session_id` | Text
+    * The unique ID for the RD Test.
+    * Auto filled by CHT integration.
+* `rdtoolkit_filter` | Text
+    * The group of RDT that users can select when provisioning RDT by using RDToolkit app.
+    * The available options can be found in the [RDToolkit app source code](https://github.com/dimagi/rd-toolkit).
+    * Assign a group name to enable all the RDT associated with the group, example: `mal_pf`.
+    * Assign individual RDT names to limit the options available for the user, separated by one space, example: “sd_bioline_mal_pf_pv carestart_mal_pf_pv”.
+
+Optional fields:
+* `rdtoolkit_api_url` | Text
+     * API URL that will receive the RDT results, sent by RDToolkit app via `POST`. See [RDToolkit app source code](https://github.com/dimagi/rd-toolkit) for more details.
+* `rdtoolkit_state` | Text
+     * The RDT status.
+     * Auto filled by the CHT integration.
+* `rdtoolkit_time_started` | Text
+     * The date and time when the RDT started.
+     * Auto filled by the CHT integration.
+* `rdtoolkit_time_resolved` | Text
+     * The date and time when the RDT result is ready to capture.
+     * Auto filled by the CHT integration.
+
+### Sample form for provisioning RDT
+![Provisioning RDT form](rdtoolkit-povision-form.png)
```

Comment:
Does this image reference work? I would have expected a path of some sort to the image.

Reply:
It turns out that the folder needs to be called the same as this file and it doesn't need path
Fixed!